README.md

```diff
@@ -1,2 +1,237 @@
-# fiware-here-backend
-Backend implementation of the integration between FIWARE and HERE Maps
+# Fiware HERE Adaptor
+
+Join the information provided by HERE and City Broker for the car navigator.
```

Comment:
Adaptor component for the FIWARE-NGSI-HERE integration for Smarter car navigation through cities.